### PR TITLE
Nep 20/bottom navigation

### PR DESCRIPTION
**Pull request by shrijan-siris**
- navigation bar is added 
- we can also use library for it as well

### CODE REVIEW

File: setup.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs')
```
Comment (syuraj) [inline setup.js]: @shrijan-siris Do you always need setup.js or it worked without setup.js also?

File: src/home/components/bottom-navigation/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { Icon } from 'react-native-elements'
+import styled from 'styled-components/native'
+import colors from '../../../config/colors'
+export default class ButtonNavigation extends Component {
```
Comment (syuraj) [inline src/home/components/bottom-navigation/index.js]: @shrijan-siris just fyi, we are using nativebase library. footer is also available there.
https://docs.nativebase.io/Components.html#footer-tabs-def-headref

**Comment by syuraj** [comment]
@shrijan-siris Can you also try writing basic tests for `Bottom Navigation` (e.g. it renders or not)? you can take app.test.js as reference.

File: setup.js (hunk 1 above)
Comment (shrijan-siris) [reply setup.js]: for now it works without setup.js , i left it for future purposes _-_ if in case

**Comment by syuraj** [comment]
@shrijan-siris if you can include screenshot, it will help us visualize without downloading and running the code.

**Comment by syuraj** [comment]
@shrijan-siris I added the screenshot for your reference so that you can add similarly
image

**Pull request merged**